fastcrypto-zkp/src/bn254/zk_login_api.rs

```diff
@@ -32,10 +32,99 @@ impl Default for ZkLoginEnv {
+        "20491192805390485299153009773594534940189261866228447918068658471970481763042".to_string(),
+        "9383485363053290200918347156157836566562967994039712273449902621266178545958".to_string(),
+        "1".to_string(),
+    ]);
+    let vk_beta_2 = g2_affine_from_str_projective(vec![
+        vec![
+            "6375614351688725206403948262868962793625744043794305715222011528459656738731"
+                .to_string(),
+            "4252822878758300859123897981450591353533073413197771768651442665752259397132"
+                .to_string(),
+        ],
+        vec![
+            "10505242626370262277552901082094356697409835680220590971873171140371331206856"
+                .to_string(),
+            "21847035105528745403288232691147584728191162732299865338377159692350059136679"
+                .to_string(),
+        ],
+        vec!["1".to_string(), "0".to_string()],
+    ]);
+    let vk_gamma_2 = g2_affine_from_str_projective(vec![
+        vec![
+            "10857046999023057135944570762232829481370756359578518086990519993285655852781"
+                .to_string(),
+            "11559732032986387107991004021392285783925812861821192530917403151452391805634"
+                .to_string(),
+        ],
+        vec![
+            "8495653923123431417604973247489272438418190587263600148770280649306958101930"
+                .to_string(),
+            "4082367875863433681332203403145435568316851327593401208105741076214120093531"
+                .to_string(),
+        ],
+        vec!["1".to_string(), "0".to_string()],
+    ]);
+    let vk_delta_2 = g2_affine_from_str_projective(vec![
+        vec![
+            "10857046999023057135944570762232829481370756359578518086990519993285655852781"
+                .to_string(),
+            "11559732032986387107991004021392285783925812861821192530917403151452391805634"
+                .to_string(),
+        ],
+        vec![
+            "8495653923123431417604973247489272438418190587263600148770280649306958101930"
+                .to_string(),
+            "4082367875863433681332203403145435568316851327593401208105741076214120093531"
+                .to_string(),
+        ],
+        vec!["1".to_string(), "0".to_string()],
+    ]);
+
+    // Create a vector of G1Affine elements from the IC
+    let mut vk_gamma_abc_g1 = Vec::new();
+    for e in vec![
+        vec![
+            "18931764958316061396537365316410279129357566768168194299771466990652581507745"
+                .to_string(),
+            "19589594864158083697499253358172374190940731232487666687594341722397321059767"
+                .to_string(),
+            "1".to_string(),
+        ],
+        vec![
+            "6267760579143073538587735682191258967573139158461221609828687320377758856284"
+                .to_string(),
+            "18672820669757254021555424652581702101071897282778751499312181111578447239911"
+                .to_string(),
+            "1".to_string(),
+        ],
+    ] {
```

Comment:
this is an old (so incorrect) vk here - i want to use explicitly as a different one (to be replaced after the ceremony) so we don't accidentally release the feature in prod cc @mskd12